library/core/class.request.php

```diff
@@ -1591,6 +1611,7 @@ public function url($path = '', $withDomain = false, $ssl = null) {
         }
 
         if (substr($path, 0, 2) == '//' || in_array(strpos($path, '://'), [4, 5])) { // Accounts for http:// and https:// - some querystring params may have "://", and this would cause things to break.
+            self::$urls[$staticKey] = ''.$path;
```

Comment:
Because according to doc comment block for path() function 
```
* @return string | null
```
it can return null. But I don't want to store null as value, but empty string.